README.rst

```diff
@@ -19,6 +19,21 @@
 Setup scripts can use pytest-runner to add setup.py test support for pytest
 runner.
 
+Deprecation Notice
+==================
+
+pytest-runner depends on deprecated features of setuptools and relies on features that break security
+mechanisms in pip. For example 'setup_requires' and 'tests_require' bypass ``pip --require-hashes``.
+See also https://github.com/pypa/setuptools/issues/1684.
+
+It is recommended that you:
+
+- Remove 'pytest-runner' from your 'setup_requires', preferably removing the `setup_requires` option.
+- Remove 'pytest' and any other testing requirements from 'tests_require', preferably removing the `setup_requires` option.
+- Use a requirement-test.txt file to hold test dependencies
+- Invoke tests with ``pip install --require-hashes requirements-test.txt && pytest``
```

Comment:
This syntax is hostile to Windows and also mutates the user's environment. Especially for a user coming from `pytest-runner`, which doesn't mutate the environment, that might be a surprise.  If a one-liner is required, I'd suggest instead `pip-run -r tests/requirements.txt -- -m pytest`. Of course, that assumes pip-run and not just pip. I think better would be to recommend a generic "select a tool to bootstrap and then run tests such as tox". Maybe link to tox or link to a section in Python packaging docs that may enumerate options and/or give the (current) best recommendation.